system/churchcore/uploadFile.php

```diff
@@ -18,13 +18,12 @@ function churchcore__uploadfile() {
     $uploader = new qqFileUploader($allowedExtensions, $sizeLimit);
   }
   catch (Exception $e) {
-    $result = "Entweder POST_MAX_SIZE und UPLOAD_MAX_SIZE erhöhen oder Zahl erniedrigen.";
+    $result = "Entweder POST_MAX_SIZE und UPLOAD_MAX_SIZE erhöhen oder Zahl erniedrigen. ";
   }
   if ($result == null) {
-    $file_dir = $files_dir . "/files/" . getVar("domain_type") . "/";
-    if ($id = getVar("domain_id")) $file_dir .= $id;
-    if (!file_exists($file_dir)) mkdir($file_dir, 0777, true);
-    $result = $uploader->handleUpload($file_dir . "/");
+    $file_dir = $files_dir . "/blobs/";
+    if (!file_exists($file_dir)) mkdir($file_dir, 0755, true);
```

Comment:
AFAIK, this is better, but might screw up shared hosting, which by definition means "no security", but is still very popular.